maturity.go

```diff
@@ -0,0 +1,60 @@
+package kindsys
+
+import (
+	"bytes"
+	"encoding/json"
+)
+
+type Maturity int8
+
+const (
+	MaturityUnknown Maturity = iota
+	MaturityMerged
+	MaturityExperimental
+	MaturityStable
+	MaturityMature
+)
+
+func (s Maturity) String() string {
+	switch s {
+	case MaturityMerged:
+		return "merged"
```

Comment:
Honestly I don't understand the need for maturity framework at all... this just mimics the properties defined in thema+kindsys today.

I think it makes more sense to:
1. not include kinds that are not used anywhere (the "merged" maturity)
2. stick "-alpha" at the end of a specific version string until it should be considered stable

but my goal here was not to revisit that structure yet!

Reply:
Ok ok, I like the suggestion you made here. The easier we can make it the better and it will increase the likelyhood of getting developers to actually use it.

Reply:
Agreed @ryantxu